python/cuml/test/test_svm.py

```diff
@@ -131,10 +131,12 @@ def compare_svm(svm1, svm2, X, y, b_tol=None, coef_tol=None,
     # Compare model parameter b (intercept). In practice some models can have
     # same differences in the model parameters while still being within
     # the accuracy tolerance.
-    if abs(svm2.intercept_) > 1e-6:
-        assert abs((svm1.intercept_-svm2.intercept_)/svm2.intercept_) <= b_tol
-    else:
-        assert abs((svm1.intercept_-svm2.intercept_)) <= b_tol
+    if not hasattr(svm2.intercept_, '__len__'):  # skip this for multiclass
```

Comment:
What type are you checking for here? Can you be more specific than just having `__len__`

Reply:
I wanted to ensure that the intercept is a scalar.  But actually, this test would be incorrect when we compare to cuML SVM classifiers, where the intercept can be array of 1 elements.  I have changed the check to ensure that the intercept is either non-array or has only a single element.